packages/cli/src/commands/deploy.ts

```diff
@@ -40,73 +40,239 @@ import {
-import { SchemaValidationFailed } from '../../util/errors';
-import purchaseDomainIfAvailable from '../../util/domains/purchase-domain-if-available';
-import confirm from '../../util/input/confirm';
-import editProjectSettings from '../../util/input/edit-project-settings';
-import {
-  getLinkedProject,
-  linkFolderToProject,
-} from '../../util/projects/link';
-import getProjectName from '../../util/get-project-name';
-import selectOrg from '../../util/input/select-org';
-import inputProject from '../../util/input/input-project';
-import { prependEmoji, emoji } from '../../util/emoji';
-import { inputRootDirectory } from '../../util/input/input-root-directory';
-import validatePaths, {
-  validateRootDirectory,
-} from '../../util/validate-paths';
-import { getCommandName } from '../../util/pkg-name';
-import { Output } from '../../util/output';
-import { help } from './args';
-import { getDeploymentChecks } from '../../util/deploy/get-deployment-checks';
-import parseTarget from '../../util/deploy/parse-target';
-import getPrebuiltJson from '../../util/deploy/get-prebuilt-json';
-import { createGitMeta } from '../../util/create-git-meta';
-import { isValidArchive } from '../../util/deploy/validate-archive-format';
-import { parseEnv } from '../../util/parse-env';
+} from '../util/errors-ts';
+import { SchemaValidationFailed } from '../util/errors';
+import purchaseDomainIfAvailable from '../util/domains/purchase-domain-if-available';
+import confirm from '../util/input/confirm';
+import editProjectSettings from '../util/input/edit-project-settings';
+import { getLinkedProject, linkFolderToProject } from '../util/projects/link';
+import getProjectName from '../util/get-project-name';
+import selectOrg from '../util/input/select-org';
+import inputProject from '../util/input/input-project';
+import { prependEmoji, emoji } from '../util/emoji';
+import { inputRootDirectory } from '../util/input/input-root-directory';
+import validatePaths, { validateRootDirectory } from '../util/validate-paths';
+import { getCommandName } from '../util/pkg-name';
+import { Output } from '../util/output';
+import { getDeploymentChecks } from '../util/deploy/get-deployment-checks';
+import parseTarget from '../util/deploy/parse-target';
+import getPrebuiltJson from '../util/deploy/get-prebuilt-json';
+import { createGitMeta } from '../util/create-git-meta';
+import { isValidArchive } from '../util/deploy/validate-archive-format';
+import { parseEnv } from '../util/parse-env';
 import { errorToString, isErrnoException, isError } from '@vercel/error-utils';
-import { pickOverrides } from '../../util/projects/project-settings';
-import { printDeploymentStatus } from '../../util/deploy/print-deployment-status';
+import { pickOverrides } from '../util/projects/project-settings';
+import { printDeploymentStatus } from '../util/deploy/print-deployment-status';
+import { help } from './help';
+import type { Command } from './help';
+
+const deployCommand: Command = {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏I think it would still make sense to have the arg definition in the `args.ts` file. Either works, though.

Reply:
done